flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/physical/batch/BatchExecSortAggRule.scala

```diff
@@ -124,6 +120,17 @@ class BatchExecSortAggRule
       } else {
         (Seq(FlinkRelDistribution.SINGLETON), RelCollations.EMPTY)
       }
+      // Remove the global agg call filters because the
```

Comment:
`BatchExecHashAggRule` should also has this logic

Reply:
Yep, thanks for the reminder ~